sdk/environments/environment.go

```diff
@@ -187,6 +191,9 @@ func (e *ApiEndpoint) AppId() (*string, bool) {
 }
 
 func (e *ApiEndpoint) Name() string {
+	if e == nil {
+		return ""
```

Comment:
potentially pedantic, but would help us diagnose when this is nil (initializing bug) vs left empty (logic bug)?

```suggestion
		return "(nil)"
```